hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/MiniDFSCluster.java

```diff
@@ -240,6 +241,10 @@ public Builder(Configuration conf) {
       }
     }
 
+    public Builder(Configuration conf, TemporaryFolder baseDir) {
```

Comment:
i don't want to add junit dependencies here; we don't know where else it is used. and test dependencies don't get exported by maven. the code will need to be given baseDir.getRoot()